src/lib/passwordGenerator.js

```diff
@@ -46,8 +46,8 @@ export const buildCharset = (options) => {
     charset.push(...[...symbols]);
   }
 
-  if (options.contentRules.similarChars === false) {
-    charset.filter((character) => similarChars.indexOf(character) >= 0);
+  if (options.allowSimilarChars === false) {
+    charset = charset.filter((char) => !similarChars.includes(char));
```

Comment:
```suggestion
    return charset.filter((char) => !similarChars.includes(char));
```
and keep charset as a const ?